operations/patch_util_test.go

```diff
@@ -311,6 +311,44 @@ func (s *PatchUtilTestSuite) TestValidatePatchCommand() {
 
 }
 
+func (s *PatchUtilTestSuite) TestProjectFieldRequired() {
+	conf, err := NewClientSettings(s.testConfigFile)
+	s.Require().NoError(err)
+
+	// Remove default flag from all projects to test that project is required
+	for i := range conf.Projects {
+		conf.Projects[i].Default = false
+	}
+
+	// Test that patchParams with a project succeeds validation
+	p := patchParams{
+		Project:     "mci",
+		Finalize:    true,
+		Uncommitted: false,
+	}
+	s.NotEmpty(p.Project, "project should be set for valid patch params")
```

Comment:
this kind of error check isn't really testing any code, since p.Project isn't a function; both this line and 336 are really just testing that go works the way we expect. It would be better to run validatePatchCommand on each of these params instead (the way you do for emptyProject at 339)